dokan_fuse/src/fusemain.cpp

```diff
@@ -424,15 +424,67 @@ int impl_fuse_context::create_directory(LPCWSTR file_name,
   return ops_.mkdir(fname.c_str(), dirmask_);
 }
 
+int impl_fuse_context::readdir_filler_set_has_files(void *buf, const char *name,
+                                      const struct FUSE_STAT *stbuf,
+                                      FUSE_OFF_T off) {
+  bool *has_files = static_cast<bool *>(buf);
+  int ret = 0;
+  if (strcmp(name, ".") && strcmp(name, "..")) {
+    //fprintf(stderr, "\tDirectory is not empty (found direntry %s).\n", name);
```

Comment:
What about this line? Remove it, uncomment it or leave as it is (commented)?
The whole file does not have any debug-prints, so the `FPRINTF`-macro defined in dokanfuse.cpp is not available.
If the dir is to be deleted it will still show `dispositionInfo->DeleteFile = 1` as the next printout in any case.